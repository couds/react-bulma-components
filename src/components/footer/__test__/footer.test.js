import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '..';

describe('Footer component', () => {
  it('Should have footer classname', () => {
    const component = renderer.create(
      <Footer>
        <p className="bd-notification is-success">
          <p>Default</p>
          <p>Container</p>
        </p>
      </Footer>,
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
