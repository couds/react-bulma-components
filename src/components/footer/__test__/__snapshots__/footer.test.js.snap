// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Should have footer classname 1`] = `
<div
  className="footer"
  style={Object {}}
>
  <p
    className="bd-notification is-success"
  >
    <p>
      Default
    </p>
    <p>
      Container
    </p>
  </p>
</div>
`;
